[DOCS] Improve document in reflection (#5593)
apache/incubator-tvm: apps/cpp_rpc/README.md

```

## Note

> Currently support is only there for Linux / Android / Windows environment and proxy mode doesn't be supported currently.

Reviewer: ```suggestion
Currently only Linux / Android / Windows environment is supported, and proxy mode is not supported.
```
